components/icon-links.jsx

```diff
@@ -2,8 +2,22 @@ var React = require('react');
 
 var IconLinks = React.createClass({
   render: function() {
+    var links = this.props.children;
+    var classes;
     return (
-      <div className="row icon-links">{this.props.children}</div>
+      <div className="row icon-links">
+        {
+          links.map(function(link, i){
+            // we either have 2 or 3 icons in a IconLinks row
+            classes = (links.length === 2 & i === 0) ?
+                      "col-sm-4 col-md-4 col-lg-4 col-sm-offset-2 col-md-offset-2 col-lg-offset-2" :
+                      "col-sm-4 col-md-4 col-lg-4";
```

Comment:
> Er wait, should `&` be `&&`? `&` is a [bitwise AND](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Bitwise_Operators#Bitwise_AND).

Ah hahahah, nice catch.  Yes it should be `&&`.

> Also, I wonder if it'd be useful during `ComponentDidMount` to check if `process.env.NODE_ENV !== 'PRODUCTION'` and log a dev warning if the number of children is an unexpected value (0, 1, or greater than 3 I think?). Could be useful both as a dev warning and as self-documenting code, but it's up to you.

Yes, I like that plan!  Will update this PR.